### PR TITLE
qsub: add page

### PR DESCRIPTION
**Pull request by fpnick**
----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines

### CODE REVIEW

File: pages/linux/qsub.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# qsub
+
+> Submits a script to the queue management system TORQUE on High Performance Computing clusters.
```
Comment (sbrl) [inline pages/linux/qsub.md]: Is there any reason why `High Performance Computing` is in title case? And is there any reason why a normal computing cluster couldn't use this tool? If not, then maybe something along the lines of `Often employed on high performance computing clusters.` would work as a separate sentence.

File: pages/linux/qsub.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+- Submit a script with default settings (depends on TORQUE settings):
+
+`qsub script.sh`
+
+- Submit a script with a specified wallclock runtime of 1h 2m 3s:
```
Comment (sbrl) [inline pages/linux/qsub.md]: What about `wallclock runtime limit`?

File: pages/linux/qsub.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+`qsub -l walltime=01:02:03 script.sh`
+
+- Submit a script that is executed on 2 nodes using 16 cores per node:
+
+`qsub -l nodes=2:ppn=16 script.sh`
+
+- Submit a script with a specified wallclock runtime of 1h 2m 3s to a specific queue. Note that different queues can have different maximum and minimum runtime limits:
```
Comment (sbrl) [inline pages/linux/qsub.md]: Same here with `wallclock runtime limit`.

File: pages/linux/qsub.md (hunk 1 above)
Comment (agnivade) [reply pages/linux/qsub.md]: I would rather remove subjective descriptions like "High Performance Computing Clusters", and just keep `Submits a script to the queue management system TORQUE`

File: pages/linux/qsub.md (hunk 3 above)
Comment (agnivade) [inline pages/linux/qsub.md]: No need to add the `walltime` option here. We can just do with adding `-q` and saying `Submit a script to a given queue.`

**Comment by agnivade** [comment]
@fpnick - tokens can still have concrete values. :wink: I have made the changes myself.

**Pull request merged**